ISSUE_TEMPLATE/01-bug-report.yml

```
@@ -0,0 +1,53 @@
+name: Bug Report
+description: Report a bug that you are experiencing.
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        When reporting bugs, please follow the guidelines in this template. This helps identify the problem precisely and thus enables contributors to fix it faster.
+        - Write a descriptive issue title above.
+        - The golden rule is to **always open *one* issue for *one* bug**. If you notice several bugs and want to report them, make sure to create one new issue for each of them.
+        - Ensure that you're on the latest release or commit on main. It's possible that the bug you're reporting has already been fixed in a more recent version of this package.
+        - Search open and closed issues to ensure it has not already been reported. If you don't find a relevant match or if you're unsure, don't hesitate to **open a new issue**.
+
+  - type: textarea
+    attributes:
+      label: Tested Versions
+      description: |
+        To properly fix a bug, we need to identify if the bug was recently introduced, or if it was always present.
+        - Please specify (if applicable) the release version you found the issue in, or the **Git commit hash** if using a development branch.
```

Nice Gary, not sure if `main` is always considered a development branch, so thought it might be best to specify main here 👍 
```suggestion
        - Please specify (if applicable) the release version you found the issue in, or the **Git commit hash** if pointing at the main branch.
```